modules/templates.js

```diff
@@ -0,0 +1,13 @@
+angular.module('pages').run(['$templateCache', function($templateCache) {
```

Comment:
Then the module `pages` should be `feincms-pages.templates`, so you need to set the `module` property of the `ngtemplates` configuration to be that. The best thing to do there, I think, is rename the `modules/pages/` folder to `modules/feincms-pages/`.

Also it should be registering a new module so it can standalone (it's an option for `ngtemplates`). So add `standalone: true` to the `ngtemplates` configuration